src/components/va-back-to-top/va-back-to-top.css

```diff
@@ -0,0 +1,101 @@
+@import '../../../node_modules/animate.css/source/sliding_entrances/slideInUp.css';
+@import '../../../node_modules/animate.css/source/sliding_exits/slideOutDown.css';
+@import '../../mixins/buttons.css';
+:host {
+  display: block;
+  bottom: 0;
+  width: 100%;
+  height: 44px;
+  margin-top: 4rem;
+  margin-bottom: 2rem;
+}
+
+div {
+  position: fixed;
+  bottom: 0;
+  display: block;
+  width: 100%;
+}
+
+.reveal-point {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  top: var(--reveal-breakpoint, 60rem);
+  left: 0;
+}
+
+button {
+  bottom: 0;
+  right: 0;
+  background-color: var(--color-gray-dark);
+  border-bottom-right-radius: 0;
+  border-top-right-radius: 0;
+  height: 44px;
+  width: 44px;
+  margin: 0;
+  margin-bottom: 0.5rem;
+  opacity: 0;
+  padding: 0;
+  pointer-events: none;
+  pointer-events: auto;
+  position: absolute;
+  transition: all 0.25s ease-in;
+}
+button:hover {
+  background-color: var(--color-black);
+}
+button.reveal {
+  pointer-events: auto;
+  opacity: 1;
+  animation: slideInUp 0.25s;
+}
+button:not(.reveal) {
+  animation: slideOutDown 0.25s;
+}
+
+button span {
+  display: none;
+}
+
+div.docked {
+  position: relative;
+  display: flex;
+  flex-direction: row-reverse;
+}
+div:not(.docked) {
+  max-width: var(--undocked-width);
+}
+
+div.docked button {
+  position: relative;
+}
+
+i {
+  font-size: 2rem;
+}
+i {
+  font-family: 'Font Awesome 5 Free';
+  font-style: normal;
+  font-weight: 900;
+}
+i::before {
+  content: '\F062'; /* fa-arrow-up*/
+}
```

Comment:
Related to the previous comment. I don't want to `@import` things from fontawesome just yet, because it isn't granular. Anywhere that fontawesome is imported, the `<style>` for the component would have every single fontawesome class. Some postcss should be able to remove all unused classes, and once that is in place I'd like to improve our component CSS.

Reply:
Might need to specify font awesome (or `fortawesome`) as a peer dependency.